governance.md

```diff
@@ -84,7 +86,7 @@ Current TOC members:
 * Rob Zienert (Netflix)
 * Travis Tomsu (Google)
 
-The technical oversight committee is initially seeded with core members from Netflix and Google. We will leave further details such as number of seats, qualifications, tenure, TBD for the time being.
+The TOC should remain with an odd number of members so as to avoid deadlock. The TOC will nominate a chair person for a 1 year term. Their duty is to run/call meetings, take meeting notes, and call votes.  
```

Comment:
Thoughts on limits? E.g. >1? Any ceiling?

Reply:
YES! Great call. We should define a ceiling - anywhere between 3 and 7 seems reasonable to me. You?

Reply:
3 to 7 also sounds good to start with. We could always increase the upper limit if needed via another vote

Reply:
3 to 7, excluding even numbers. In the event that someone from TOC leaves prematurely, the SC should be proactive in filling the spot so we're back to an odd number of members. Thoughts?

Reply:
Sounds good to me.

Reply:
Adding this language.